### ISSUE TITLE
Unable to update ?

### ISSUE THREAD

**Issue by Anthonymcqueen21** [commixproject/commix#638]
I have a problem updating Commix i had Parrot OS as my prefered platform but after many attempts to clone the latest repository it for some reason does not update. So i decided to install Kali-Linux as my new platform to pentest and it worked initially and now i have the same problem...

[warning] Python version 3.9.2 detected. You are advised to use Python version 2.7.x.
                                      __           
   ___   ___     ___ ___     ___ ___ /\_\   __  _   
 /`___\ / __`\ /' __` __`\ /' __` __`\/\ \ /\ \/'\  v3.2-stable
/\ \__//\ \/\ \/\ \/\ \/\ \/\ \/\ \/\ \ \ \\/>  </  
\ \____\ \____/\ \_\ \_\ \_\ \_\ \_\ \_\ \_\/\_/\_\ https://commixproject.com
 \/____/\/___/  \/_/\/_/\/_/\/_/\/_/\/_/\/_/\//\/_/ (@commixproject)

+--
Automated All-in-One OS Command Injection Exploitation Tool
Copyright © 2014-2021 Anastasios Stasinopoulos (@ancst)
+--

(!) Legal disclaimer: Usage of commix for attacking targets without prior mutual consent is illegal. It is the end user's responsibility to obey all applicable local, state and federal laws. Developers assume no liability and are not responsible for any misuse or damage caused by this program.

**[warning] You haven't updated commix for more than 51 days!**
[critical] Invalid target URL has been given.

So long story short should i continue to use the latest git repository or should i use a master repository to use this tool effectively ?

**Comment by stasinopoulos** [comment]
@Anthonymcqueen21 preferably, you can download commix by cloning the Git repository, in order to be able to update it via using the `--update` flag.

**Comment by Anthonymcqueen21** [comment]
Okay thank you i believe i did it correctly but i have to do it manually each time. Is there a other way where it will be automatically updated whenever i choose to use it for testing purposes ?
